src/test/java/bio/terra/service/load/LoadDaoUnitTest.java

```diff
@@ -161,4 +162,39 @@ private void testLoadCandidates(
     assertThat("right number of running", candidates.getRunningLoads(), hasSize(running));
     assertThat("right number of not_tried", candidates.getCandidateFiles(), hasSize(notTried));
   }
+
+  @Test
+  void unlockLoadByDatasetIdOnly() {
+    final String loadTag1 = "loadTag1";
+    final String loadTag2 = "loadTag2";
+    final String flightX = "flightIdX";
+    final String flightY = "flightIdY";
+    final UUID datasetId = dataset.getId();
+    final LoadLockKey loadTag1_dataset = new LoadLockKey(loadTag1, datasetId);
+    final LoadLockKey loadTag2_dataset = new LoadLockKey(loadTag2, datasetId);
```

Comment:
Sonar flags these names as not conforming to our code standard. It's not a huge deal but it would be nice to avoid adding sonar warnings if possible
```suggestion
    final LoadLockKey loadTag1Dataset = new LoadLockKey(loadTag1, datasetId);
    final LoadLockKey loadTag2Dataset = new LoadLockKey(loadTag2, datasetId);
```